.ado/jobs/desktop.yml

```diff
@@ -12,16 +12,18 @@ jobs:
       - template: ../variables/vs2019.yml
       #5059 - Disable failing or intermittent tests (IntegrationTestHarness,AsyncStorage,WebSocket,Logging).
       #5265 - WebSocketModuleIntegrationTest::WebSocketModule_Ping fails for Release
+      # windesktop is no longer bundling - Disable RNTesterIntegrationTests until fixed.
```

Comment:
>until fixed [](start = 76, length = 11)

Please add work item number that tracks this work.

Reply:
Actually,that statement seems to not apply anymore (see description).

Will remove.